packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -70,7 +70,7 @@ export const TabsItem = compose<TabsItemType>({
       },
     };
 
-    const buttonRef = useViewCommandFocus(componentRef);
+    const buttonRef = Platform.OS == 'macos' ? componentRef : useViewCommandFocus(componentRef);
```

Comment:
nit: I think this should also be `===`
Also @chiuam fyi since she's investigating ref issues on macOS

Reply:
fixed!

Reply:
here's the bug I filed for this: https://office.visualstudio.com/OC/_workitems/edit/5382203